src/core_plugins/console/public/src/es.js

```diff
@@ -16,6 +16,11 @@ export function send(method, path, data) {
   var wrappedDfd = $.Deferred();
 
   console.log("Calling " + path);
+  var isGETRequest = /get/i.test(method)
```

Comment:
Can we ensure the entire string is `get` and camelcase the variable name?

`var isGetRequest = /^get$/i.test(method)`